source/MRMesh/MRObject.h

```diff
@@ -238,7 +238,7 @@ class MRMESH_CLASS Object : public ObjectChildrenHolder
     /// returns the amount of memory this object occupies on heap
     [[nodiscard]] MRMESH_API virtual size_t heapBytes() const;
 
-    /// signal about xf changing, triggered in setXf and setWorldXf,  it is called for children too
+    /// signal about xf changing, triggered in setXf, setWorldXf, addChildBefore and addChild,  it is called for children too
```

Comment:
it is better to note that the signal for addChild is called not for this object, but for children